addons/skin.estuary/xml/DialogAddonSettings.xml

```diff
@@ -123,6 +123,63 @@
 					<param name="label" value="" />
 				</include>
 			</control>
+			<control type="group">
+				<description>Currently-playing emulator name and icon</description>
+				<visible>Player.HasGame + String.IsEqual(Window(addonsettings).Property(Addon.Type),kodi.gameclient)</visible>
+				<left>1480</left>
+				<width>310</width>
+				<top>382</top>
+				<height>360</height>
+				<control type="group">
+					<left>52</left>
+					<control type="image">
+						<description>Background image</description>
+						<left>-20</left>
+						<right>-20</right>
+						<top>-20</top>
+						<bottom>-20</bottom>
+						<texture border="40">buttons/dialogbutton-nofo.png</texture>
+					</control>
+					<control type="label">
+						<description>Label for "Playing with:" text</description>
+						<top>14</top>
+						<height>20</height>
+						<font>font16</font>
+						<shadowcolor>text_shadow</shadowcolor>
+						<label>$LOCALIZE[35271]</label>
+						<align>center</align>
+					</control>
+					<control type="label">
+						<description>Emulator name</description>
```

Comment:
why not add text "<scroll>" to avoid cutting long names?

Reply:
Good idea, updated the PR.

I also borrowed the XML here from my work on the Savestate Manager, which can also improve from a `<scroll>` tag. I'll just cram that change into this PR.